### PR TITLE
Modify default configuration file to prevent fails during LPVS service start

### PR DESCRIPTION
**Pull request by m-rudyk**
# Description

Changed default configuration in application properties to prevent issues with Default launch:
- added table properties configuration;
- added default token (requred change in order to operate with real repo);

Fixes # (207)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -14,9 +14,9 @@ license_conflict=db
 
 # GitHub settings
 # Corresponding env. variable LPVS_GITHUB_LOGIN
-github.login=
+github.login=test
```
Comment (o-kopysov) [inline src/main/resources/application.properties:17]: Disagree. Why do we need to set default values?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -42,3 +42,9 @@ spring.datasource.url=jdbc:mysql://localhost:3306/lpvs
 spring.datasource.username=
 spring.datasource.password=
 
+# Custom DB components name configuration
+app.table.detectedLicenseName=detected_license
+app.table.detectedLicenseSchema=soshub
+app.table.diffFileName=code_licenses
+app.table.pullRequestsName=lpvs_pull_requests
+app.table.queueName=queue
```
Comment (o-kopysov) [inline src/main/resources/application.properties:50]: Why do we need these properties for the issue #206  fix?

**Review by o-kopysov — CHANGES_REQUESTED** [review]
Please check my comments

**Comment by m-rudyk** [comment]
> Please check my comments
after conversation, I've put additional verification on application property set. Overall, Spring application failed to load due to absence of token. 
Table configuration is set to prevent another failure when injecting app property

**Review by o-kopysov — APPROVED** [review]
I have no issues with LPVS run.

```
$ java -jar lpvs-*.jar

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::               (v2.7.10)

2023-09-08 14:05:28.682  INFO 24816 --- [           main] com.lpvs.LicensePreValidationSystem      : Starting LicensePreValidationSystem v1.1.0 using Java 11.0.6 on o-kopysov05 with PID 24816 (D:\Git\GitHub\LPVS-Github\LPVS\target\lpvs-1.1.0.jar started by o.kopysov in D:\Git\GitHub\LPVS-Github\LPVS\target)
2023-09-08 14:05:28.688  INFO 24816 --- [           main] com.lpvs.LicensePreValidationSystem      : No active profile set, falling back to 1 default profile: "default"
2023-09-08 14:05:30.021  INFO 24816 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-09-08 14:05:30.247  INFO 24816 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 191 ms. Found 5 JPA repository interfaces.
2023-09-08 14:05:34.881  INFO 24816 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 7896 (http)
2023-09-08 14:05:34.914  INFO 24816 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-09-08 14:05:34.915  INFO 24816 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.73]
2023-09-08 14:05:35.144  INFO 24816 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-09-08 14:05:35.144  INFO 24816 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 6340 ms
2023-09-08 14:05:35.683  INFO 24816 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-09-08 14:05:35.855  INFO 24816 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.6.15.Final
2023-09-08 14:05:36.367  INFO 24816 --- [           main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2023-09-08 14:05:36.708  INFO 24816 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-09-08 14:05:37.871  INFO 24816 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-09-08 14:05:37.933  INFO 24816 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2023-09-08 14:05:40.230  INFO 24816 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2023-09-08 14:05:40.255  INFO 24816 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-09-08 14:05:41.605  INFO 24816 --- [           main] com.lpvs.service.LPVSLicenseService      : LICENSES: loaded 5 licenses from DB.
2023-09-08 14:05:41.645  INFO 24816 --- [           main] com.lpvs.service.LPVSLicenseService      : LICENSE CONFLICTS: loaded 3 license conflicts from DB.
2023-09-08 14:05:41.649  INFO 24816 --- [           main] com.lpvs.service.LPVSDetectService       : License detection scanner: scanoss
2023-09-08 14:05:42.829  INFO 24816 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 7896 (http) with context path ''
2023-09-08 14:05:42.852  INFO 24816 --- [           main] com.lpvs.LicensePreValidationSystem      : Started LicensePreValidationSystem in 15.027 seconds (JVM running for 16.173)

```

File: src/main/java/com/lpvs/service/LPVSGitHubService.java [hunk 3]
```diff
@@ -88,12 +88,15 @@ public LPVSGitHubService(@Value("${" + GITHUB_LOGIN_PROP_NAME + "}") String GITH
...
-    @Profile("!test")
     private void checks() throws Exception {
         if (this.GITHUB_AUTH_TOKEN.isEmpty()) {
             log.error(GITHUB_AUTH_TOKEN_ENV_VAR_NAME + "(" + GITHUB_AUTH_TOKEN_PROP_NAME + ") is not set.");
             exitHandler.exit(-1);
         }
+        if (this.GITHUB_LOGIN.isEmpty()) {
+            log.error(GITHUB_LOGIN_ENV_VAR_NAME + "(" + GITHUB_LOGIN_PROP_NAME + ") is not set.");
+            exitHandler.exit(-1);
+        }
```
Comment (tiokim) [inline src/main/java/com/lpvs/service/LPVSGitHubService.java:99]: LPVS can run without `GITHUB_LOGIN`. Please do not include this code.

**Review by o-kopysov — APPROVED** [review]
LGTM

**Review by tiokim — APPROVED** [review]
LGTM!

**Pull request merged**